cpp/src/IceGrid/Activator.cpp

```diff
@@ -641,20 +641,50 @@ Activator::activate(const string& name,
     }
 
     vector<gid_t> groups;
+#ifdef _AIX
+    char* grouplist = getgrset(pw->pw_name);
```

Comment:
We need to free grouplist after we are done with it.
http://www.regatta.cs.msu.su/doc/usr/share/man/info/ru_RU/a_doc_lib/libs/basetrf1/getgrset.htm

Reply:
Fixed